app/views/videos/show.html.haml

```diff
@@ -14,6 +14,8 @@
               There are no reviews
         %p= @video.description
         .actions
+          = link_to("+ My Queue", queue_items_path(video_id: @video.id), method: :post, :class => 'btn btn-default')
```

Comment:
Its a small detail, but I would not use `:class => "..."` and `method: :verb` together - as you are using 2 different hash notations, one is for Ruby 1.9+ and the class one is older. Nowadays the new syntax is much preferred.